### ISSUE TITLE
Conformance to WCAG standard: key board accessibility

### ISSUE THREAD

**Issue by sandhraprakash** [mendixlabs/mendix-dynamic-table#13]
Is there a plan to add keyboard accessibility(On tab or arrow keys) to move around inside the matrix (empty cell and filled cell)

**Comment by sandhraprakash** [comment]
@JelteMX I saw that you have enhanced the widget to accommodate keyboard accessibility in v1.4.0. 
But for some reason, I am still not able to access the widget via keyboard. it skips right over the control to the next one. Tab index of the control is set to 0 by default.
Am I missing something over here?
